include/RAJA/policy/cuda/MemUtils_CUDA.hpp

```diff
@@ -297,186 +299,226 @@ cudaDeviceProp& device_prop()
 }
 
 
+//! Struct with the maximum theoretical occupancy of the device
```

Comment:
Would it be helpful for the various structs defined in this file to have their fields initialized to defaults to indicate a nominal use case or uninitialized state?

Reply:
Maybe? I didn't want to move the named initializers out into the namespace scope though.

Reply:
OK. Whatever you think is best.

Reply:
That is true, I'll think about how to do this a bit more.

Reply:
Ok, I did this and it did make the code better.